html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+    <h1>Coding Form!</h1>
+  </head>
+  <body>
+    <div></div>
```

Comment:
Rather than having empty divs to make line breaks, you can put the elements as a child of the `div` to get the same effect, which would avoid a bunch of empty `div` elements in your code.